From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Williamson <chris@chrisaw.io>
Date: Wed, 05 Feb 2026 00:00:00 +0000
Subject: [PATCH 3/4] Remove MusicFX from build

MusicFX is not included in GrapheneOS GSI builds. Remove it from
the handheld system packages and generic system image to prevent
build failures when the source is excluded via manifest.

Change-Id: I0000000000000000000000000000000000000002
---
 target/product/generic/Android.bp | 1 -
 target/product/handheld_system.mk | 1 -
 2 files changed, 2 deletions(-)

diff --git a/target/product/generic/Android.bp b/target/product/generic/Android.bp
--- a/target/product/generic/Android.bp
+++ b/target/product/generic/Android.bp
@@ -817,7 +817,6 @@ android_system_image {
                 "MediaProviderLegacy", // base_system
                 "MmsService", // handheld_system
                 "MtpService", // handheld_system
-                "MusicFX", // handheld_system
                 "NetworkStack", // base_system
                 "ONS", // telephony_system
                 "PacProcessor", // handheld_system
diff --git a/target/product/handheld_system.mk b/target/product/handheld_system.mk
--- a/target/product/handheld_system.mk
+++ b/target/product/handheld_system.mk
@@ -61,7 +61,6 @@ PRODUCT_PACKAGES += \
     ManagedProvisioning \
     MmsService \
     MtpService \
-    MusicFX \
     NetworkLocation \
     PacProcessor \
     preinstalled-packages-platform-handheld-system.xml \
-- 
2.34.1

